Add statements and principles (#1)
Mjolner-ICSA25/odense-manifesto: principles.md

# Principles made during working session

1. To achieve relevant industrial impact, the projects need to bridge the gab between research and product development
2. Align the education with industry & academic needs by teaching fundamental theories through practical applications to ensure future og industry & academia is secured for innovative real-world solutions

> 3. People over profits - To begin your study research, you need a strong motivation: Make sure you choose a path og (personal) growth, where you can also make some money

Reviewer: As far as I remember, the focus of this one was more in the direction of using science and knowledge for the good. I.e. to steer research in the direction of benefitting humans, instead of merely focusing on optimizing for making more money. Analog to not developing nuclear weapons for war, but for humanity. 

Maybe we could express that best like this:

```suggestion
3. People over profits - To begin your study research, you need a strong motivation: Make sure you choose a path that may benefit humans, where you can also make some money
```

Or maybe something like `...a path that may benefit all humans...`,  or `...a path that may benefit humanity...`, etc.

4. Multiple made
   1. Ground the research in relevant industrial concerns before creating an original solution
   2. Ground the research in relevant industrial concerns of today or the near-term future before creating an original solution
5. Make use of scientific knowledge as driver for mutual learning
6. Multiple made
   1. The academic system should consider industry relevant research/grants for academic tenure or for academic promotions
   2. Academia should be able to understand industry problems, develop solutions and then generalize as a common knowledge
   3. The students should put in real world scenarios where they can work with industry during their coursework

> 7. To bring true value, the training of (future) software architects should add to `foundational knowledge`, `critical thinking` more importantly that the `latest technologies`

Reviewer 2: ```suggestion
7. To bring true value, the training of (future) software architects should add to `foundational knowledge`, `critical thinking` more importantly than the `latest technologies`
```

I think we meant a comparison here :)

We could also add the sub statement to (7)
